dev/docker-compose.yml

```diff
@@ -80,6 +80,39 @@ services:
       - pg
       - testdb
 
+  openssl:
+    image: svagi/openssl:latest
+    entrypoint:
```

Comment:
can this be transferred to something like:
```
entrypoint: ["some", "commands"]
```

Reply:
i think that even if the line is long it will be more readable than the current state

Reply:
Are you sure? this is the resulting line:
```
entrypoint: ["openssl", "req", "-newkey", "rsa:2048", "-days", "365", "-nodes", "-x509", "-config", "tmp/conf/tls.conf", "-extensions", "v3_ca", "-keyout", "tmp/conf/nginx.key", "-out", "tmp/conf/nginx.crt"]
```

Reply:
yuck. will this work?
```
entrypoint: [
  "openssl", 
  "req", 
  "-newkey", 
  "rsa:2048", 
  "-days", 
  "365", 
  "-nodes", 
  "-x509", 
  "-config", 
  "tmp/conf/tls.conf", 
  "-extensions", 
  "v3_ca", 
  "-keyout", 
  "tmp/conf/nginx.key", 
  "-out", 
  "tmp/conf/nginx.crt"
]
```

Reply:
done